frontend/test/__support__/e2e/cypress.js

```diff
@@ -1,9 +1,36 @@
 require("cypress-grep")();
 
+import addContext from "mochawesome/addContext";
 import "@testing-library/cypress/add-commands";
 import "cypress-real-events/support";
 import "@cypress/skip-test/support";
 import "@percy/cypress";
 import "./commands";
 
 Cypress.on("uncaught:exception", (err, runnable) => false);
+
+const titleToFileName = title => title.replace(/[>]/g, "");
```

Comment:
This might be nitpicking, but I have the same feedback as in the previous PR attempt:
https://github.com/metabase/metabase/pull/21727#discussion_r908397740

Reply:
What would be your suggestion ?

Reply:
I think it's sensible enough to move `titleToFileName` inside the `Cypress.on("test:after:run" ...` block.